kde-apps/knights/knights-9999.ebuild

```diff
@@ -19,21 +19,19 @@ KEYWORDS=""
 IUSE="speech"
 
 DEPEND="
-	>=dev-qt/qtbase-${QTMIN}:6[concurrent,dbus,gui,network,widgets]
+	>=dev-qt/qt5compat-${QTMIN}:6
+	>=dev-qt/qtbase-${QTMIN}:6[concurrent,gui,widgets]
 	>=dev-qt/qtsvg-${QTMIN}:6
 	>=kde-apps/libkdegames-${PVCUT}:6
-	>=kde-frameworks/kcompletion-${KFMIN}:6
-	>=kde-frameworks/kconfig-${KFMIN}:6
 	>=kde-frameworks/kconfigwidgets-${KFMIN}:6
 	>=kde-frameworks/kcoreaddons-${KFMIN}:6
 	>=kde-frameworks/kcrash-${KFMIN}:6
 	>=kde-frameworks/kdbusaddons-${KFMIN}:6
-	>=kde-frameworks/ki18n-${KFMIN}:6
```

Comment:
It would be very weird for this to be dropped, need to double check...

Reply:
Its not explicitly listed, but I do see its required through KTextWidgets.

Reply:
that's why looking at the linking is important:
```
$ lddtree /usr/bin/knights
knights => /usr/bin/knights (interpreter => /lib64/ld-linux-x86-64.so.2)
[...]
    libKF6I18n.so.6 => /usr/lib64/libKF6I18n.so.6
```

So, definitely missing dependency in upstream's cmake.

Reply:
https://invent.kde.org/games/knights/-/merge_requests/24